mnefun/_mnefun.py

```diff
@@ -2011,6 +2018,9 @@ def do_preprocessing_combined(p, subjects, run_indices):
                 projs.extend(pr)
             else:
                 warnings.warn('Only %d EOG events!' % eog_events.shape[0])
+            if p.plot_pca:
+                h = plot_projs_topomap(pr, info=raw.info, show=False)
+                h.savefig(eog_proj[:-4] + '.png', format='png', dpi=120)
```

Comment:
IIRC there are also empty-room projectors, can you check?

Reply:
There are erm projectors but not considered here. Do you want SSP topomaps for them?

Reply:
Done